kubernetes/configure

```diff
@@ -1,4 +1,11 @@
-#!/bin/bash -x
+#!/bin/bash
+
+profile=${1:-stsstack}
+
+if [[ $profile == "stsstack" ]]; then
```

Comment:
We can drop these checks since stsstack is going away by the end of the week, lets make it compatible with ps6 only now.